src/main/resources/hudson/plugins/jira/JiraSite/config.jelly

```diff
@@ -10,7 +10,7 @@
     <f:checkbox title="${%Use HTTP authentication instead of normal login}" field="useHTTPAuth" />
   </f:invisibleEntry>
   <f:entry>
-    <f:checkbox title="${%Use Bearer authentication instead of Basic authentication}" field="useBearerAuth" description="${%Note: Bearer authentication is only supported in Jira Server, for Jira Cloud leave this unchecked}" />
+    <f:checkbox title="${%Use Bearer authentication instead of Basic authentication}" field="useBearerAuth" description="${%site.useBearerAuth}" />
```

Comment:
I just tested the last build but the description text did not show up. So it seems the `description` needs to be on the [<f:entry>](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/entry.jelly) tag instead of [<f:checkbox>](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/checkbox.jelly). After that change the description showed up for me fine.

Reply:
Should I move just description on entry level or do you think that it's better to move everything (e.g. title and description) on entry level?

Reply:
For some reason my comment hasn't been published.. Should I move just description on entry tag or also field and title attributes?

Reply:
Just the description, since [field](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/checkbox.jelly#L62) and [title](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/checkbox.jelly#L65) are valid for checkbox, only `description` does not exist in that jelly (for unknown reason for me..)

Reply:
Done 👍